jira/jira-v1.asciidoc

```diff
@@ -0,0 +1,94 @@
+== Jira REST Configuration
+
+This documentation describes aspects of the Jira REST `jira-v1.json` file configuration such as the authentication methods, data crawled and retrieved, pagination information, variables used, and endpoints used. Terminology is also provided as a reference.
+
+The list of data the REST connector crawls using the jira REST configuration is:
+
+* Projects 
+* Comments
+* Worklogs  
+* Attachments
+
+
+== Authentication methods
+
+The Jira REST configuration supports:
+
+* Basic Authentication using the username and password from an Atlassian account. For more information, see link:https://developer.atlassian.com/cloud/confluence/basic-auth-for-rest-apis/[Basic auth for REST APIs | Atlassian Developer^].
+
+* API Token. For information about how to create a new API token, see link:https://id.atlassian.com/manage/api-tokens[API Tokens | Atlassian ID^].
+
+
+== Supported crawl options
+
+The Jira REST configuration supports the following crawl options:
+
+* Full crawl
+
+* Recrawl that uses the `rely on the strayContentDeletion` parameter
+
+
+== Pagination information
+
+This recipe is configured to use pagination by batch size. 
+
+* The `${LW_INDEX_START}` variable is set to zero because the Atlassian pagination is zero-index-based. 
+
+* The `${LW_BATCH_SIZE}` variable is set to `20`, but can be adjusted to any value.
+
+* The pagination `stop` condition is set to a `results` response object set `[]`. Pagination stops when this condition is met.
+
+
+== Variables used
+
+The Jira REST configuration variables used are:
+
+* `${LW_BATCH_SIZE}` - This variable is used to set the `limit` query parameter, which controls the number of results that are returned in the response for both `blogpost` and `pages`.
+
+* `${LW_INDEX_START}` - This variable is used to set the `start` query parameter, which is used to traverse the pagination. Jira pagination is zero-index-based. This means the first page number is 0.
```

Comment:
Change to: Add Used with pagination feature. This variable is used to set the `startAt` query parameter, which is used to traverse the pagination. Jira pagination is zero-index-based. This means the first page number is 0.